addons/point_of_sale/static/src/xml/Screens/TicketScreen/TicketScreen.xml

```diff
@@ -6,11 +6,21 @@
             <div class="screen-content">
                 <div class="controls">
                     <div class="buttons">
-                        <button t-if="showNewTicketButton" class="highlight" t-on-click="createNewOrder">New Order</button>
-                        <button class="discard" t-on-click="trigger('close-screen')">Discard</button>
+                        <t t-if="env.isMobile">
+                            <button t-att-class="{'search': !state.showSearchBar,'arrow-left': state.showSearchBar}" t-on-click="toggleSearchBar">
+                                <span class="search-icon"><i class="fa" t-att-class="{'fa-search': !state.showSearchBar,'fa-arrow-left': state.showSearchBar}"/></span>
+                                <t t-if="!state.showSearchBar">
+                                    Search
```

Comment:
Not sure that we want this button on bottom... On other screens like products page, the search bar is on the top.